src/jobs/deploy-ecs.yml

```diff
@@ -1,9 +1,60 @@
 description: >
   This job deploys an application to aws elastic container service.
 
+parameters:
+  # Applicaiton parameters
+  environment:
+    type: string
+    default: 'node'
+    description: >
+      Pick the language that the application is written in.
+      This value determines what type of executor should be used.
+  version:
+    type: string
+    default: 'latest'
+    description: >
+      Pick the version of the application to deploy to Dockerhub and ECS.
+  
+
 executor: node/default
```

Comment:
Is this supposed to be dynamically set?

Reply:
if it was statically set then every application would be deployed as the same version everytime.